### PR TITLE
Split user log and runtime log into seperate files

### PR DESCRIPTION
**Pull request by Binyang2014**
This change contains following chanages:
1. Collect DockerCotainerLog to runtime.docker.pai.error
2. Collect YarnContainerLog to runtime.yarn.pai.error
3. Collect user log to user.pai.error

Testing Done:
Just test the basic function for redirection command

### CODE REVIEW

File: src/rest-server/src/templates/yarnContainerScript.mustache [hunk 1]
```diff
@@ -26,14 +26,19 @@ export PAI_DEFAULT_FS_URI={{ hdfsUri }}
...
+
 # YARN kill container will send SIGTERM(15) and SIGKILL(9)
 function sigterm_handler()
 {
   printf "[DEBUG] SIGTERM received, job may be killed by YARN\n"
   killed_by_yarn=true
   exit 143
-}
-
+} 1 >> $RUNTIME_YARN_ERR_LOG 2>> $RUNTIME_YARN_LOG
```
Comment (mzmssg) [inline src/rest-server/src/templates/yarnContainerScript.mustache]: `1 >> $RUNTIME_YARN_LOG 2>> $RUNTIME_YARN_ERR_LOG`?

File: src/rest-server/src/templates/yarnContainerScript.mustache [hunk 2]
```diff
@@ -62,8 +67,9 @@ function exit_handler()
   fi
 
   exit $rc
-}
+} 1 >> $RUNTIME_YARN_ERR_LOG 2>> $RUNTIME_YARN_LOG
```
Comment (mzmssg) [inline src/rest-server/src/templates/yarnContainerScript.mustache]: the same

File: src/rest-server/src/templates/yarnContainerScript.mustache (hunk 1 above)
Comment (Binyang2014) [reply src/rest-server/src/templates/yarnContainerScript.mustache]: It's a mistake, thanks for point that

**Comment by yqwang-ms** [comment]
To avoid breaking change, let's first merge all our PR (Runtime, Launcher, RestServer, etc) into errorspec branch. And only until all are ready in Phase 2, we will merge errorspec branch to merge branch. 
@Binyang2014 , @sunqinzheng , is that ok or any better suggestions?

**Comment by Binyang2014** [comment]
Per yuqi request, close this PR and create a new to errorspec branch

**Pull request closed without merging**